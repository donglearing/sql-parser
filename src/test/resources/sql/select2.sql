select a, b, c from T where a = 1 and b = '15-10-2017' or c in (1, 2, 3)