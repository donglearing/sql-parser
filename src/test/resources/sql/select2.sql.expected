SELECT a, b, c FROM T WHERE a = 1 AND b = '15-10-2017' OR c IN (1, 2, 3)